utils_cv/detection/model.py

```diff
@@ -0,0 +1,283 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import numpy as np
+from typing import List, Tuple, Union
+from pathlib import Path
+from functools import partial
+
+from PIL import Image
+import cv2
+import torch
+import torch.nn as nn
+from torchvision import transforms
+from torchvision.models.detection import fasterrcnn_resnet50_fpn
+from torchvision.models.detection.faster_rcnn import FastRCNNPredictor
+from torch.utils.data import Dataset, DataLoader
+import matplotlib.pyplot as plt
+
+from .references.transforms import RandomHorizontalFlip, Compose, ToTensor
```

Comment:
Not sure if "references" is the best name. Bet let's keep for now, can always change later.